src/butil/memory/scoped_vector.h

```diff
@@ -6,6 +6,9 @@
 #define BUTIL_MEMORY_SCOPED_VECTOR_H_
 
 #include <vector>
+#if __cplusplus >= 201103L  // >= C++11
```

Comment:
no need to add this, c++11 is enabled by default

Reply:
I have fixed it.